homeassistant/util/dt.py

```
@@ -197,6 +197,9 @@ def q_n_r(first: int, second: int) -> Tuple[int, int]:
         """Return quotient and remaining."""
         return first // second, first % second
 
+    if type(date) is dt.datetime and not date.tzinfo:
```

This shouldn't be fixed in util/dt.py but instead in the template filter

---

We should create a new function that wraps `get_age` and set it here https://github.com/home-assistant/core/blob/dev/homeassistant/helpers/template.py#L971

---

I will update the PR

---

As i discussed in the dev_core channel  it looks like in other filters that exceptions is silenced and the filter should return the input. Should we handle it like that. The user in some cases will not notice that anything went wrong. Like in strptime where wrong input returns the input string. I suggest that it returns None or empty string. This might be a new issue.